providers/file/file.go

```diff
@@ -120,9 +121,18 @@ func (f *File) Watch(cb func(event interface{}, err error)) error {
 			}
 		}
 
-		w.Close()
+		f.w.Close()
 	}()
 
 	// Watch the directory for changes.
-	return w.Add(fDir)
+	return f.w.Add(fDir)
+}
+
+func (f *File) UnWatch() error {
```

Comment:
Can we add a comment to this method? Also, would it be better to just name it as Unwatch?

Reply:
Please verify the comment on the method